### PR TITLE
Streamline README

### PR DESCRIPTION
**Pull request by cbrnr**
I've added some links and improved/clarified some things here and there. Let me know if the latest master of MNE is required, because I thought a simple `pip install mne` would be sufficient.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,40 +7,33 @@
...
-Next to `numpy`, `scipy`, and `matplotlib` that are included in the standard
-anaconda distribution, you will need to install the following dependencies
-to be able to use `mne_bids`:
+We recommend the [Anaconda](https://www.anaconda.com/download/) Python distribution.
+Besides `numpy`, `scipy`, `matplotlib`, and `pandas` (which are included in the standard
+Anaconda installation), you will need to install MNE using the `pip` tool:
 
-    $ pip install pandas
-    $ pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/master
+    $ pip install -U mne
```
Comment (sappelhoff) [inline README.md]: indeed, MNE master is required at least until 0.17 is released.

Comment (cbrnr) [reply README.md]: OK, I'll add a note and revert this line then.

**Comment by teonbrooks** [comment]
there are some dependencies on the latest version of mne. I think we would need to specify 
`mne >= 0.17`

**Comment by jasmainak** [comment]
> Looks good to me, but @jasmainak wants to change the version numbers before we merge PRs again. Perhaps this can be done in this PR actually ...

okay this is done. We're good to merge PRs again :-)

**Comment by jasmainak** [comment]
Merged, thanks @cbrnr !

**Pull request merged**